source/qml/Library/RendererQml/AdaptiveCardQmlRenderer.cpp

```diff
@@ -643,16 +681,27 @@ namespace RendererQml
 
 		if ((input->GetMin() == input->GetMax() && input->GetMin() == 0) || input->GetMin() > input->GetMax())
 		{
-			input->SetMin(INT_MIN);
+			input->SetMin(0);
 			input->SetMax(INT_MAX);
 		}
+		if (input->GetValue() == std::nullopt)
+		{
+			uiNumberInput->Property("property bool hasDefaultValue", "false");
+		}
+		else
+		{
+			uiNumberInput->Property("property bool hasDefaultValue", "true");
+			uiNumberInput->Property("property int defaultValue", std::to_string(input->GetValue().value()));
+		}
 		if (input->GetValue() < input->GetMin())
 		{
 			input->SetValue(input->GetMin());
+			uiNumberInput->Property("property int defaultValue", std::to_string(input->GetMin().value()));
```

Comment:
These should be readonly property.

Reply:
And this will change once we change [this ](https://github.com/AdaptiveCards-Desktop/AdaptiveCards/pull/122/files#diff-7ec96e6b73f993ffc36bf3e8fa668b1adb508e09a5e436ac452ec48d57c9c4d9L637)logic.